Updated supported data formats supported by Tidb Lightning (#7921) (#7940)
pingcap/docs: tidb-lightning/tidb-lightning-faq.md

1 row in set (0.01 sec)
```

## What kinds of data source formats are supported by TiDB Lightning?

TiDB Lightning supports:

> - Importing files exported by [Dumpling](/dumpling-overview.md), CSV files, and [Apache Parquet files generated by Amazon Aurora](/migrate-aurora-to-tidb.md).

Reviewer: ```suggestion
- Importing files exported by [Dumpling](/dumpling-overview.md), CSV files, and [Apache Parquet files generated by Amazon Aurora](/tidb-lightning/migrate-from-csv-using-tidb-lightning.md).
```

> - Importing files exported by [Dumpling](/dumpling-overview.md), CSV files, and [Apache Parquet files generated by Amazon Aurora](/migrate-from-aurora-using-lightning.md).

Reviewer 2: ```suggestion
- Importing files exported by [Dumpling](/dumpling-overview.md), CSV files, and [Apache Parquet files generated by Amazon Aurora](/migrate-from-aurora-using-lightning.md). 
```

> - Importing files exported by [Dumpling](/dumpling-overview.md), CSV files, and [Apache Parquet files generated by Amazon Aurora](/tidb-lightning/migrate-from-aurora-using-lightning.md).

Reviewer 2: ```suggestion
- Importing files exported by [Dumpling](/dumpling-overview.md), CSV files, and [Apache Parquet files generated by Amazon Aurora](/migrate-from-aurora-using-lightning.md).
```

> - Importing files exported by [Dumpling](/dumpling-overview.md), CSV files, and [Apache Parquet files generated by Amazon Aurora](/tidb-lightning/migrate-from-csv-using-tidb-lightning.md).

Reviewer: ```suggestion
- Importing files exported by [Dumpling](/dumpling-overview.md), CSV files, and [Apache Parquet files generated by Amazon Aurora](/tidb-lightning/migrate-from-aurora-using-tidb-lightning.md).
```

> - Importing files exported by [Dumpling](/dumpling-overview.md), CSV files, and [Apache Parquet files generated by Amazon Aurora](/tidb-lightning/migrate-from-csv-using-tidb-lightning.md).

Reviewer 2: ```suggestion
- Importing files exported by [Dumpling](/dumpling-overview.md), CSV files, and [Apache Parquet files generated by Amazon Aurora](/tidb-lightning/migrate-from-aurora-using-lightning.md).
```
